samples/msal-node-samples/auth-code/package.json

```diff
@@ -18,6 +18,7 @@
     "express": "^4.17.1"
   },
   "devDependencies": {
+    "express-session": "^1.17.2",
```

Comment:
This should be in the regular `dependencies` list, given its necessary for the app at runtime.